Trophy.md

```diff
@@ -0,0 +1,31 @@
+# 🏆 Bugs & CVEs 
+
+The showcase of bugs and CVEs found by *UTopia*-based fuzzers serves a variety of purpose:
+- Show the effect of the fuzzing automation approach (i.e.,*UTopia* can help find more bugs in the projects that are already registered and have been being fuzz tested in OSS-Fuzz)
+- Provide insight into making unit tests that can work well with *UTopia* (i.e., Fuzzing friendly unit tests)
+- Increase the visibility of various fuzz drivers so that people can learn or improve fuzz driver composing strategies
+
+Note that The merged fuzz drivers are different from the original ones generated by *UTofia* because they are reviewed and modified by the community.

```

Comment:
typo: The merged -> the merged
typo: UTofia -> UTopia

Reply:
👍